generators/app/index.js

```diff
@@ -131,6 +131,14 @@ export default class extends Generator {
 			} else {
 				this.config.set("defaultTheme", "sap_fiori_3");
 			}
+
+			// set qunit coverage file
+			if (semver.gte(props.frameworkVersion, "1.113.0")) {
+				// set qunit-coverage file
```

Comment:
duplicated comment. Better add a reference to the middleware README which contains this minimum version requirement.

Reply:
done